File: tests/java_static/src/test/java/myairtable/TestHarness.java
package myairtable;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

/** Proves the Gradle harness compiles static/java and runs JUnit 5 tests. */
class TestHarness {

  @Test
  void staticRuntimeIsOnTheClasspath() {
    assertEquals("0.0.1-dev", MyAirtableRuntimeInfo.VERSION);
  }
}

File: tests/kotlin_static/src/test/kotlin/myairtable/TestHarness.kt
package myairtable

import kotlin.test.Test
import kotlin.test.assertEquals

/** Proves the Gradle harness compiles static/kotlin and runs JUnit 5 tests. */
class TestHarness {
    @Test
    fun staticRuntimeIsOnTheClasspath() {
        assertEquals("0.0.1-dev", MyAirtableRuntimeInfo.VERSION)
    }
}
